docs/api.rst

```diff
@@ -10589,32 +10662,85 @@ Downloading a database snapshot
       Content-Type: application/zip
 
 
-   :statuscode 200: Snapshot was downloaded successfully.
+   :statuscode 200: Snapshot was deleted successfully.
    :statuscode 400: Provided JSON is in some way malformed.
-   :statuscode 409: No user is currently logged in. No snapshot data found for the given timestamp. No permissions to write in the given directory. Check error message.
+   :statuscode 409: No user is currently logged in. No snapshot found for the specified timestamp.Check error message.
    :statuscode 500: Internal rotki error.
 
 
-Deleting a database snapshot
-=================================
+.. http:patch:: /api/(version)/snapshots/(timestamp)
+
+   Doing either a PATCH on this endpoint will update the snapshot of the specified timestamp with the provided payload.
```

Comment:
`either` shouldn't be here

Reply:
It deletes the snapshot and writes a new one provided by the user for that particular timestamp.
The issue describes how the functionality works.

Will make it more descriptive

Reply:
Yeah yeah, I understand what you meant. The comment was from the reader's perspective